setup.nsi

```diff
@@ -61,7 +61,7 @@ Function InstModeChanged
 SetShellVarContext CURRENT
 ${IfNotThen} ${Silent} ${|} StrCpy $InstDir "${S_DEFINSTDIR_USER}" ${|}
 ${If} $InstMode > 0
-	SetShellVarContext ALL
+	SetShellVarContext CURRENT
```

Comment:
Somehow this `SetShellVarContext ALL` was making %APPDATA% to be in `C:\ProgramData` from `C:\Users\{user}\AppData\Roaming` which we do not want.

Reply:
That's not the typical meaning of that option. Generally, the only reason you'd install software for the "current user only" is if you didn't have admin privileges. "Install for all users" usually just means the following:

* Install in a world-readable-but-only-admin-writable location (usually `Program Files`), rather than in a single-user-R/W-access location (usually the user's profile directory).
* Put the uninstall info and any machine-wide configuration data in the `HKEY_LOCAL_MACHINE` root registry key, rather than in the `HKEY_CURRENT_USER` key.
* Put Start Menu shortcuts under the All Users Start Menu, rather than the Current User Start menu.

Actual application data, generated by or for individual users, should always be stored in the user's profile (as is now being done). `All Users`/`ProgramData`, like `HKEY_LOCAL_MACHINE` is a suitable location for things like license information (if the app is licensed on a per-install basis), configuration options an administrator wants to enforce for all users, and so on.

Reply:
Changed it back